playbooks/roles/jenkins/templates/amazon-cloudwatch-agent.json.j2

```diff
@@ -0,0 +1,53 @@
+{
+  "agent": {
+    "metrics_collection_interval": 60,
+    "region": "eu-west-1",
+    "logfile": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log",
```

Comment:
Does the metrics need to be logged to here? Could we put the log file in `/var/log` instead?

Reply:
Good question.

It's actually the default location as detailed here:
<img width="1423" alt="screen shot 2019-03-04 at 15 12 27" src="https://user-images.githubusercontent.com/3469840/53742200-031dad00-3e90-11e9-953a-bd3e6f205afb.png">


Should I remove that line in the agent section, or move the logfile to `/var/log`?